.github/workflows/nightly.yml

```diff
@@ -58,3 +59,51 @@ jobs:
     - name: Fuzz long test
       working-directory: ${{github.workspace}}/build
       run: ctest -C ${{matrix.build_type}} --output-on-failure -L "fuzz-long"
+
+  adapter-test:
+    name: Test - Adapters on Ubuntu
+    strategy:
+      matrix:
+        adapter: [L0]
+        build_type: [Debug, Release]
+        compiler: [{c: gcc, cxx: g++}]
+
+    runs-on: devcloud-pvc
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Install pip packages
+      run: python3 -m pip install -r third_party/requirements.txt
+
+    - name: Download DPC++
+      run: |
+        wget -O ${{github.workspace}}/dpcpp_compiler.tar.gz https://github.com/intel/llvm/releases/download/sycl-nightly%2F20230626/dpcpp-compiler.tar.gz
+        tar -xvf ${{github.workspace}}/dpcpp_compiler.tar.gz
+
+    - name: Setup DPC++
+      run: |
+        source ${{github.workspace}}/dpcpp_compiler/startup.sh
+
+    - name: Configure CMake
+      run: |
+        cmake \
+        -B${{github.workspace}}/build \
+        -DCMAKE_C_COMPILER=${{matrix.compiler.c}} \
+        -DCMAKE_CXX_COMPILER=${{matrix.compiler.cxx}} \
+        -DUR_ENABLE_TRACING=ON \
+        -DUR_DEVELOPER_MODE=OFF \
+        -DCMAKE_BUILD_TYPE=${{matrix.build_type}} \
+        -DUR_BUILD_TESTS=ON \
+        -DUR_FORMAT_CPP_STYLE=OFF \
+        -DUR_BUILD_ADAPTER_${{matrix.adapter}}=ON \
+        -DUR_DPCXX=${{github.workspace}}/dpcpp_compiler/bin/clang++
+
+    - name: Build
+      run: |
+        cmake --build ${{github.workspace}}/build -j $(nproc)
+
+    - name: Test
+      working-directory: ${{github.workspace}}/build
+      run: |
+        ctest -C ${{matrix.build_type}} --output-on-failure -L "conformance"
```

Comment:
pls run it like it's run here: https://github.com/oneapi-src/unified-runtime/blob/adapters/.github/workflows/cmake.yml#L241

pls, also add the same comment